cmd/observe/observe.go

```diff
@@ -463,7 +461,7 @@ func handleArgs(ofilter *observeFilter, debug bool) (err error) {
 	return nil
 }
 
-func runObserve(conn *grpc.ClientConn, ofilter *observeFilter) error {
+func runObserve(conn *grpc.ClientConn, ofilter *observeFilter, debug bool, stderr io.Writer) error {
```

Comment:
Instead of overloading this function, what about creating a new one that builds the request and pass `req` to this `runObserve`? You may build the request and print it before calling `runObserve`.